moulinette/interfaces/api.py

```diff
@@ -406,13 +404,11 @@ def logout(self, profile):
 
         """
         s_id = request.get_cookie("session.id")
-        try:
-            # We check that there's a (signed) session.hash available
-            # for additional security ?
-            # (An attacker could not craft such signed hashed ? (FIXME : need to make sure of this))
-            s_secret = self.secrets[s_id]
-            request.get_cookie("session.tokens", secret=s_secret, default={})[profile]
-        except KeyError:
+        # We check that there's a (signed) session.hash available
+        # for additional security ?
+        # (An attacker could not craft such signed hashed ? (FIXME : need to make sure of this))
+        s_secret = self.secrets[s_id]
```

Comment:
Here I think the point was also to catch the case where `s_id` ain't in `self.secrets` ... Though I'm not sure if that makes any sense, it's been a long time ...